packages/matter.js/src/node/ServerNode.ts

```diff
@@ -57,7 +62,15 @@ export class ServerNode<T extends ServerNode.RootEndpoint = ServerNode.RootEndpo
     constructor(config: Partial<Node.Configuration<T>>);
 
     constructor(definition?: T | Node.Configuration<T>, options?: Node.Options<T>) {
-        super(Node.nodeConfigFor(ServerNode.RootEndpoint as T, definition, options));
+        const config = Node.nodeConfigFor(ServerNode.RootEndpoint as T, definition, options);
+
+        // Enable reachableChanged event on the BasicInformation cluster of Root endpoint if reachable attribute is defined
+        if (config.type.deviceType === RootEndpoint.deviceType && config.basicInformation?.reachable !== undefined) {
```

Comment:
But this.enable or such was not possible in initialize. How do I need to do it then to enable the event?

Reply:
Ok I can set the event using this.events.xxyzz= but this do not enables if for the eventList on the cluster. I would need some more advise on how to move it correctly to initialize

Reply:
Cool, it looks like we need to tweak ValidatedElements to look at the actual instance rather than solely the EndpointType.  Let me take a look at that.